Content.Server/Doors/Systems/AirlockSystem.cs

```diff
@@ -34,15 +34,18 @@ private void OnPowerChanged(EntityUid uid, AirlockComponent component, PowerChan
                 appearanceComponent.SetData(DoorVisuals.Powered, args.Powered);
             }
 
+            if (!TryComp(uid, out DoorComponent? door))
```

Comment:
Don't remember if you were at last meeting but the preference is to use the generics versions now.